src/lib/blockchain/metamask.ts

```diff
@@ -66,6 +68,7 @@ export class Metamask {
         this.account as string
       );
       this.trader = new Trader(this.web3 as Web3, this.account as string);
+      this.frog = new Frog(this.web3 as Web3, this.account as string);
```

Comment:
Since the core team does not have control over this contract we will need an extra `try / catch` clause.

This will be helpful if a community contract decided to destroy the contract (which could cause a whijte screen in the game 😓 )

I.e

```
try {
 this.frog = new Frog(this.web3 as Web3, this.account as string);
} catch (e) {
console.error("Unable to initialise frog contract", e)
}
```